src/styles/main.scss

```
@@ -0,0 +1,2 @@
+@import "./blocks/variables-clock";
+@import "./blocks/page-clock";
```

```suggestion
@import "./variables";
@import "./blocks/clock";
```

Bad file structure and naming